demos/common/mqtt_demo_helpers/mqtt_demo_helpers.c

```diff
@@ -205,26 +192,15 @@ static bool mqttSessionEstablished = false;
  * This function is an implementation the #BackoffAlgorithm_RNG_t interface type
  * of the backoff algorithm library API.
  *
- * @return The generated random number. This function ALWAYS succeeds
- * in generating a random number.
- */
-static int32_t prvGenerateRandomNumber();
-
-/**
- * @brief Seed the random number generator used for exponential backoff
- * used in connection retry attempts.
+ * The PKCS11 module is used to generate the random number as it allows access
+ * to a True Random Number Generator (TRNG) if the vendor platform supports it.
+ * It is recommended to use a device-specific unique random number generator so
+ * that probability of collisions from devices in connection retries is mitigated.
  *
- * The PKCS11 module is used to seed the random number generator
- * as it allows access to a True Random Number Generator (TRNG) if the
- * vendor platform supports it.
- * It is recommended to use a device-specific unique entropy source
- * to seed the random number generator so that probability of collisions
- * from devices in connection retries is mitigated.
- *
- * @return pdPASS if seeding the randon number generator was successful;
- * otherwise pdFAIL for failure.
+ * @return pdPASS if generating the randon number was successful; otherwise pdFAIL
```

Comment:
What does this `@return` statement refer to?  It doesn't match the implementation of `prvGenerateRandomNumber()`.  Also, spelling: __random__.

Reply:
Good catch. Have fixed the comment and the typo.